LiteFarmOrg/LiteFarm
# LF 2502 - Temperature reading component

To Test:

1. Add data sensor readings from these two CSV files. make changes in the sensor_id column of sensor_readings to map your sensor_ids present in the sensor table. 

[sensor.csv](https://github.com/LiteFarmOrg/LiteFarm/files/9074325/sensor.csv)
[sensor_reading.csv](https://github.com/LiteFarmOrg/LiteFarm/files/9074326/sensor_reading.csv)

2. Navigate to http://localhost:3000/sensor/<location_id_of_sensor>/readings
3. check if you can find this page.

<img width="1111" alt="Screen Shot 2022-07-08 at 10 42 15 AM" src="https://user-images.githubusercontent.com/20675885/178050046-6371d0f7-2ec9-4212-a67a-edc161e2d9b5.png">

4. For the Requirements to test. please check the description under.
https://lite-farm.atlassian.net/browse/LF-2502
